lib/client.js

```diff
@@ -1,32 +1,40 @@
 'use strict'
 
+module.exports = Client
+
 const orm = require('./connection')
 const uuid = require('uuid')
+const Token = require('./token')
+
+function Client (opts) {
+  this.id = opts.id
+  this.client_id = opts.client_id
+  this.client_secret = opts.client_secret
+  this.name = opts.name
+  this.homepage = opts.homepage
+  this.description = opts.homepage
+  this.callback = opts.callback
+  this.created = opts.created
+  this.deleted = opts.deleted
+}
 
-class Client {
-  constructor (opts) {
-    this.id = opts.id
-    this.client_id = opts.client_id
-    this.secret = opts.secret
-    this.name = opts.name
-    this.homepage = opts.homepage
-    this.description = opts.homepage
-    this.callback = opts.callback
-    this.created = opts.created
-    this.deleted = opts.deleted
-  }
+Client.prototype.generateToken = function (email) {
+  return Token.objects.create({
+    client_id: this.id,
+    user_email: email
+  })
```

Comment:
One note about providing `generateToken` as a method on `Client`: ormnomnom will accept promises for any values in the "filter", "create", or "update" args, so you can do:

```
Token.objects.create({
  client: Client.objects.get({client_id: clientId}),
  user_email: 'some@email.com'
})
```

If the `Client` object lookup fails, the promise returned by `Token.objects.create` will forward that error through.